dataactcore/scripts/raw_sql/link_broken_subaward_contracts.sql

```diff
@@ -10,44 +10,215 @@ WITH unlinked_subs AS
-    awarding_agency_name = aw_dap.awarding_agency_name,
-    funding_agency_code = aw_dap.funding_agency_code,
-    funding_agency_name = aw_dap.funding_agency_name,
-    award_description = aw_dap.award_description,
-    naics_description = aw_dap.naics_description
+    unique_award_key = ldap.unique_award_key,
+    award_amount = ldap.total_obligated_amount,
+    action_date = bdap.action_date,
+    fy = 'FY' || fy(bdap.action_date),
+    awarding_agency_code = ldap.awarding_agency_code,
+    awarding_agency_name = ldap.awarding_agency_name,
+    awarding_sub_tier_agency_n = ldap.awarding_sub_tier_agency_n,
+    awarding_office_code = ldap.awarding_office_code,
+    awarding_office_name = ldap.awarding_office_name,
+    funding_agency_code = ldap.funding_agency_code,
+    funding_agency_name = ldap.funding_agency_name,
+    funding_sub_tier_agency_co = ldap.funding_sub_tier_agency_co,
+    funding_sub_tier_agency_na = ldap.funding_sub_tier_agency_na,
+    funding_office_code = ldap.funding_office_code,
+    funding_office_name = ldap.funding_office_name,
+    awardee_or_recipient_uei = ldap.awardee_or_recipient_uei,
+    awardee_or_recipient_legal = ldap.awardee_or_recipient_legal,
+    dba_name = ldap.vendor_doing_as_business_n,
+    ultimate_parent_unique_ide = ldap.ultimate_parent_unique_ide,
+    ultimate_parent_uei = ldap.ultimate_parent_uei,
+    ultimate_parent_legal_enti = ldap.ultimate_parent_legal_enti,
+    legal_entity_country_code = ldap.legal_entity_country_code,
+    legal_entity_country_name = ldap.legal_entity_country_name,
+    legal_entity_address_line1 = ldap.legal_entity_address_line1,
+    legal_entity_city_name = ldap.legal_entity_city_name,
+    legal_entity_state_code = ldap.legal_entity_state_code,
+    legal_entity_state_name = ldap.legal_entity_state_descrip,
+    legal_entity_zip = CASE WHEN ldap.legal_entity_country_code = 'USA'
+                            THEN ldap.legal_entity_zip4
+                            ELSE NULL
+                       END,
+    legal_entity_congressional = ldap.legal_entity_congressional,
+    legal_entity_foreign_posta = CASE WHEN ldap.legal_entity_country_code <> 'USA'
+                                      THEN ldap.legal_entity_zip4
+                                      ELSE NULL
+                                 END,
+    place_of_perform_city_name = ldap.place_of_perform_city_name,
+    place_of_perform_state_code = ldap.place_of_performance_state,
+    place_of_perform_state_name = ldap.place_of_perfor_state_desc,
+    place_of_performance_zip = ldap.place_of_performance_zip4a,
+    place_of_perform_congressio = ldap.place_of_performance_congr,
+    place_of_perform_country_co = ldap.place_of_perform_country_c,
+    place_of_perform_country_na = ldap.place_of_perf_country_desc,
+    award_description = bdap.award_description,
+    naics = ldap.naics,
+    naics_description = ldap.naics_description,
+
+    -- File F Prime Awards
```

Comment:
Feels out of place, especially given all of these are at the prime award level. I know this was written here beforehand when we were originally making the File F/Subaward table. I think we were using these to keep track of "stuff we pull from FSRS, stuff we pull from Transactions, stuff we use in File F, and stuff we just have for tracability/tracking." I dunno if we want to make those distinctions again now or after we reapproach these tables again if that's slated for anytime soon (probably alongside when we update USAS's subaward approach, so we can then update it alongside the Ultimate Subaward Crosswalk spreadsheet). For now we can leave these in as a reminder to review this.